commit d03094649d39949a30513bf3ffb03a28fecbccd8
Author: Adam Yi <ayi@janestreet.com>
Date:   Wed Mar 8 03:11:47 2023 -0500

    hurd: fix build of tst-system.c
    
    We made tst-system.c depend on pthread, but that requires linking with
    $(shared-thread-library). It does not fail under Linux because the
    variable expands to nothing under Linux, but it fails for Hurd.
    
    I tested verified via cross-compiling that "make check" now works
    for Hurd.
    
    Signed-off-by: Adam Yi <ayi@janestreet.com>
    Reviewed-by: Adhemerval Zanella  <adhemerval.zanella@linaro.org>

diff --git a/stdlib/Makefile b/stdlib/Makefile
index cd32f53b56..5e38f0e6a2 100644
--- a/stdlib/Makefile
+++ b/stdlib/Makefile
@@ -281,6 +281,7 @@ LDLIBS-test-on_exit-race = $(shared-thread-library)
 LDLIBS-tst-canon-bz26341 = $(shared-thread-library)
 LDLIBS-tst-arc4random-fork = $(shared-thread-library)
 LDLIBS-tst-arc4random-thread = $(shared-thread-library)
+LDLIBS-tst-system = $(shared-thread-library)
 
 LDLIBS-test-dlclose-exit-race = $(shared-thread-library)
 LDFLAGS-test-dlclose-exit-race = $(LDFLAGS-rdynamic)
